src/components/Process/Create/Sidebar/BasicConfig.tsx

```diff
@@ -216,14 +216,12 @@ export const BasicConfig = () => {
         <Alert status='error'>
           <AlertIcon />
           <AlertDescription fontSize='sm'>
-            <Trans i18nKey='calendar.max_duration_exceeded'>
-              The selected duration exceeds the maximum allowed duration of {{ maxDuration }} days for your plan. Reduce
-              the voting length or{' '}
-              <Button variant='link' onClick={() => openModal('subscription')}>
-                upgrade your plan
-              </Button>
-              .
-            </Trans>
+            <Trans
+              i18nKey='calendar.max_duration_exceeded'
+              values={{ maxDuration }}
+              components={{ a: <Link as={RouterLink} to={Routes.dashboard.settings.support} /> }}
+              defaults='Duration exceeds your plan’s {{ maxDuration }}-day limit. Reduce the voting length, or <a>contact us</a> if you need more days.'
+            />
```

Comment:
No need to change it since it would regenerate translations, but just so you know, you can also do it this way:

```suggestion
            <Trans i18nKey='calendar.max_duration_exceeded'>
              Duration exceeds your plan’s {{ maxDuration }}-day limit. Reduce the voting length, or <Link as={RouterLink} to={Routes.dashboard.settings.support}>contact us</Link> if you need more days.
            </Trans>
```

This way is more readable, though you do not control the name of the component being added (which will result in some numeric component like `<2></2>` in the translation file).